### DISCUSSION TITLE
Importing entry inside existing DB

### DISCUSSION THREAD

**Discussion by MembershipExcel** [keepassxreboot/keepassxc#7955] [Q&A]
Hi
Is there a way to add entry to an existing DB ?

While I know we can import from a CSV file, each time it like creating a new DB.

I'm a bit sick of typing again and again those information while the DB already exist.

Information are
DB name, password, path,ect. 

Thank you

**Comment by droidmonkey** [comment]
Not yet, there are various complications to importing into an existing database including name conflicts and merging with existing entries.
